templates/blocks/author_profile.twig

```diff
@@ -6,9 +6,21 @@
 				<h5 rel="author" class="author-block-info-name">
 					<a href="{{ post.author.link }}" itemprop="name">{{ post.author.name }}</a>
 				</h5>
+
+				{% set author_bio_readmore %}
+					...  &nbsp;<a href="{{ post.author.link }}">{{ __( 'Read More', 'planet4-master-theme' ) }}</a>
+				{% endset %}
+
+				{% if ( fn('wp_is_mobile') ) %}
+					{% set author_bio_word_limit = 24 %}
+				{% else %}
+					{% set author_bio_word_limit = 50 %}
+				{% endif %}
+
 				<p class="author-block-info-bio" itemprop="description">
-					{{ fn('wpautop', fn('the_author_meta', 'description', author.id))|e('wp_kses_post')|raw }}
+					{{ fn('wp_trim_words', fn('wpautop', post.author.description ), author_bio_word_limit, author_bio_readmore)|e('wp_kses_post')|raw }}
```

Comment:
Is there a function that allows to specify a number of characters and that then cuts off, but doesn't do so within a word? That way we can have a predictable amount of characters, regardless of how big the words are. Using `wp_trim_words` there is quite a big range of sizes the result could have. That could be problematic in languages that use a lot of long words like Turkish.

Reply:
The `substr($author_bio, 0, strpos($author_bio, ' ', 320));` will work in this case.